templates/corporate/development-community.md

```diff
@@ -226,12 +226,30 @@ This will search the full history of all public streams for `<your
 keyword(s)>`, including messages sent before you joined and messages
 on public streams you’re not subscribed to.
 
+## Linking to GitHub issues and pull requests
+
+We've set up [custom linkifiers](/help/add-a-custom-linkifier) so that
+it's easy to link to issues and pull requests for the various Zulip
+project repositories on GitHub when composing a message or [creating a
+topic](/help/starting-a-new-topic). For example, to link to issue 1234 in:
+
+- the [Zulip server and web application][server-web-repository] repository,
+  use `#1234`.
+- the [Zulip mobile client][mobile-repository] repository, use `#M1234`.
+- the [Zulip terminal client][terminal-repository] repository, use `#T1234`.
+- the [Zulip desktop client][desktop-repository] repository, use `#D1234`.
+
```

Comment:
I wonder if a table would work better here?

Reply:
Maybe? But we'd need to figure out some CSS styling. I think the only other table in the landing pages is on the plans page where we compare Zulip to other applications, and that formatting wouldn't work. We definitely don't have any tables in the markdown files for the landing pages.

<details>
<summary>Rough draft of reformatting markdown as a table</summary>

![Screenshot from 2023-03-07 20-12-08](https://user-images.githubusercontent.com/63245456/223528168-8a0e63de-db7f-434b-bb18-8adcd4f1a389.png)
</details>

Reply:
Maybe we could format the lead-in phrase to the bullet points better so the points could just be:
- Repository name: `linkifier text`

Reply:
Looks nice to me; I don't think we need a Markdown table.